pkg/scaffold/v2/manager/config.go

```diff
@@ -76,9 +76,7 @@ spec:
     matchLabels:
       control-plane: controller-manager
       controller-tools.k8s.io: "1.0"
```

Comment:
Can we remove this annotation from all manifests ?

Reply:
> controller-tools.k8s.io: "1.0"

We can definitely remove these labels. But this may end up causing we duplicate almost all of the shared files, because most manifests files have that label. So I'd like to do it in a followup PR, since it's a cleanup.

Reply:
Filed #764 to track